lib/src/editor.dart

```diff
@@ -204,7 +204,7 @@ class HtmlEditorState extends State<HtmlEditor> {
         isChildOfList = true;
       } else if (node.nodeName === 'SPAN' && node.style != undefined) {
         // check for color, bold, etc in style:
-        if (node.style.fontWeight === 'bold') {
+        if (node.style.fontWeight === 'bold' || node.style.fontWeight === '700') {
```

Comment:
why not compare it is more than `700`

Reply:
The bool value is only 700, if it is larger it will be another type.